terasoluna-gfw-common-libraries/terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/codelist/CodeListInterceptor.java

```diff
@@ -96,49 +85,13 @@ public boolean preHandle(HttpServletRequest request,
             return true;
         }
 
-        Locale locale = RequestContextUtils.getLocale(request);
-        logger.debug("locale for I18nCodelist is '{}'.", locale);
```

Comment:
- [ ] Please be aware that debug logs will be output as many as the number of CodeList.

Reply:
Because of  the I18nCodeList bean definitions defined the individual locale map, the locale also resolved respectively. If the difference appeared between each locale maps, maybe it is an application configuration bug. Debug log usually use for debug, so I think it is a bit issue.

Reply:
@btkitsunedukam  Yes, a little issue for guideline or migration guide.

Reply:
@yoshikawaa
I will add warning for that to guideline.